articles/support/tickets.md

```diff
@@ -19,23 +19,45 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
 
 ## Open a new ticket
 
-1. From [Support Center](${env.DOMAIN_URL_SUPPORT}), click on the **Contact Support** button. If you don't see this button, then you do not have access to support (only paying and trial customers have). In this case you can use our [Community](https://community.auth0.com/) instead (click **Ask our Community**).
+1. From [Support Center](${env.DOMAIN_URL_SUPPORT}), click on the **Open Ticket** button. If you don't see this button, then you do not have access to support (only paying and trial customers have access to Support). In this case you can use our [Community](https://community.auth0.com/) instead (click **Ask our Community**).
 ![Support Center](/media/articles/support/open-ticket.png)
-1. Choose the **Affected Account** from the dropdown menu.
-1.  Under **What can we help you with?** select the type of issue that best fits your case.
+1. Choose the **Affected Tenant** from the dropdown menu.
+1. Under **Issue Type** select the type of issue that best fits your case.
+* Public Cloud Support Incident
+* Appliance Support Incident 
+* Enhancement/Feature Request
+* Billing or Payment 
+* Compliance/Legal
```

Comment:
The enumeration of issue types shouldn't he listed here, since it is described in more detail below.

Reply:
Removed list.